DiagnosticsExtension/Models/ConnectionStringValidator/ConnectionStringValidationResult.cs

```diff
@@ -40,9 +40,20 @@ public enum ResultStatus
             MsiFailure,
             EmptyConnectionString,
             MalformedConnectionString,
-            UnknownError
+            UnknownError,
+            ManagedIdentityCredentialMissing,
+            FullyQualifiedNamespaceMissed,
+            SystemAssignedManagedIdentity,
+            UserAssignedManagedIdentity,
+            ManagedIdentityCredential,
```

Comment:
This result status does not communicate any status or error.

Reply:
fixed